File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegStore.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegStore
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOStore.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClStore store) 
        {
            return DAO.DAOStore.sqlSelect(store);
        }
        public  static int crear(ClStore store)
        {
            return DAO.DAOStore.sqlInsert(store);
        }
        public  static int eliminar(ClStore store)
        {
            return DAO.DAOStore.sqlDelete(store);
        }
        public  static int actualizar(ClStore store)
        {
            return DAO.DAOStore.sqlUpdate(store);
        }
    }
}

File: Sakila_Net/VB_SakilaLib/Neg/NegStore.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegStore
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOStore.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( store as ClStore) as Boolean
                        Return DAO.DAOStore.SelectOne(store)     
                    End Function
                    Public  Shared Function crear( store as ClStore) as Integer
                        Return DAO.DAOStore.SqlInsert(store)     
                    End Function
                    Public  Shared Function eliminar( store as ClStore) as Integer
                        Return DAO.DAOStore.SqlDelete(store)     
                    End Function
                    Public  Shared Function actualizar( store as ClStore) as Integer
                        Return DAO.DAOStore.SqlUpdate(store)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
